30-day-warranty.md

```diff
@@ -0,0 +1,75 @@
+# Title
+
+30 Day Warranty
+
+# Problem
+
+A team developing a component which is used throughout an organization is
+resisting to accept or rejects contributions (feature requests) and as a result
+blocks progress or is disrupted by frequent escalations.
+
+- TBD: link to pattern "setting clear expectations for contributing code"
+
+# Context
+
+Teams depends on another team accepting their contributions so that a component
+produced by the receiving team can be used by the contributing team. The
+receiving team does not have the resources, knowledge, permission, inclination 
+to write the contributed component.
+
+# Forces
+
+- There is distrust of contributions due to a past history of cheating: teams
+  submitted half finished contributions and subsequently filed requests for 
+  fixes that make it ready for use in production.
+- If code is contributed from outside the team, the team has the natural 
+  suspicition that the other team does not know how to write code that would
+  meet the teams expectations.
+- Direction of loyalty: ?
+- There is a natural aversion to taking responsibility for code not written
+  by oneself.
+- Contributed needs to be heavily rewritten before being accepted into the
+  codebase.
+- There is the fear of the contributors not being available for support with
+  fixing bugs after the time on contribution.
+
```

Comment:
Additional forces may be:
- Teams fear contributed code will lead to high(er) maintenance costs but do not know how to control for that
- Receiving teams may fear that teaching others how to contribute code will expose technical debt in their system and that visibility may be damaging 
- Receiving teams may not believe that they will get acceptable code no matter how much mentoring they provide
- Either team may not feel confident in measuring risks or certifying that they are mitigated in a contribution